projects/genesis-lrt/index.js

```diff
@@ -1,15 +1,44 @@
 const ADDRESSES = require('../helper/coreAssets.json')
 
 async function tvl(api) {
-  if (api.timestamp > 1715181240) return {}
   const staked = await api.call({ abi: 'uint256:totalAssets', target: '0xf073bAC22DAb7FaF4a3Dd6c6189a70D54110525C' })
```

Comment:
the token suppy in other chains are excluded from totalAssets value here?

Reply:
No sir! I needed help with that. To the ethereum TVL we need to subtract the other L2 TVL Value

Reply:
the way we do it for other projects is, count the tvl only as part of chain where users interact with the contract

Reply:
But can't we count the bridged assets as the tvl for the other chains?
Cause in reality the bridge is the interaction

Reply:
sorry no, we dont treat bridge as interaction for other chains

Reply:
ok

Reply:
Let's do this then. We take off all the other chains and leave just ethereum!
We just show Genesis TVL has Genesis [Native Restaking] and show InceptionLRT [Isolated Restaking].
Can we do this?

Reply:
@g1nt0ki Bumping this up again sir

Reply:
Perfect! Really appreciate this sir!

Reply:
Hey @g1nt0ki! When do you think we can have this live?

Reply:
@g1nt0ki Any news on this?
This has been open for a while now

Reply:
@g1nt0ki Should be good now

Reply:
@g1nt0ki Also please don't forget to change the frontend